concourse/pipelines/linux-image-build.yaml

```diff
@@ -527,6 +527,34 @@ jobs:
       gcs_url: ((.:gcs-url))
       google_cloud_repo: "stable"
       build_date: ((.:build-date))
+# Image E2E test
+# centos
+- name: e2e-test-centos-7
+  plan:
+  - get: guest-test-infra
+  - get: compute-image-tools
+  - get: centos-7-gcs
+    passed: [build-centos-7]
+    trigger: false
+    params:
+      skip_download: "true"
+  - load_var: source-version
+    file: centos-7-gcs/version
+  - load_var: gcs-url
+    file: centos-7-gcs/url
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: e2e-test-centos-7
+    file: guest-test-infra/concourse/tasks/image-e2etest.yaml
+    vars:
```

Comment:
where is testbinarypath?

Reply:
what i meant by this was that the 'testbinarypath' argument was missing in the pipeline config.